### PR TITLE
Update License.txt

### PR DESCRIPTION
**Pull request by Zolar4ever**
Nice and amazing project.

### CODE REVIEW

**Comment by Alimikazeem** [comment]
Great project ever been part of

**Comment by Samuxzi** [comment]
I love this project. I am grateful

File: License.txt [hunk 1]
```diff
@@ -11,4 +11,5 @@ Unless required by applicable law or agreed to in writing, software
 distributed under the License is distributed on an "AS IS" BASIS,
 WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
-limitations under the License.
\ No newline at end of file
+limitations under the License.
+Good and ok
```
Comment (Samuxzi) [inline License.txt:15]: Great project

**Comment by Nathan076** [comment]
Choice coin is the future of finance

**Comment by mdsamuel** [comment]
This is wonderful project, can't wait for launching

**Pull request closed without merging**